app/api/entities/specs/entitySavingManager.spec.ts

```diff
@@ -80,11 +83,12 @@ describe('entitySavingManager', () => {
 
     describe('new entity', () => {
       it('should create an entity without attachments', async () => {
+        const mockedUser = userInContextMock.mockEditorUser();
         const entity = { title: 'newEntity', template: template1Id };
         const { entity: savedEntity } = await saveEntity(entity, { ...reqData });
 
         expect(savedEntity.permissions).toEqual([
-          { level: 'write', refId: 'userId', type: 'user' },
+          { level: 'write', refId: mockedUser._id?.toString(), type: 'user' },
```

Comment:
Under what condition can the _id of mockedUser._id can be undefined?  I'm concerned about the IDE configuration and typing throughout these commit.